src/renderer/components/settings/NotificationSettings.tsx

```diff
@@ -246,6 +246,33 @@ export const NotificationSettings: FC = () => {
             </Text>
           }
         />
+
+        <Stack direction="horizontal" gap="condensed" alignItems="center">
+          <Text as="label" htmlFor="notificationVolume">
```

Comment:
The rendering of this input looked a bit out of place compared to the other settings.

Can you try and copy the pattern used by controls such as `Zoom`, `Open Links`, `Theme`.